tools/annotation-processor/src/main/resources/META-INF/services/javax.annotation.processing.Processor

```diff
@@ -0,0 +1,73 @@
+#
```

Comment:
3x copiright? to nejako zle vygenerovalo

Reply:
aha, teď jsem to dal pryč a při commitu tam přidal další :) z toho souboru je nějaký zmatený ;)

Reply:
upravil jsem to přes GitHub a už je to správně ;)